src/security/content-security-policy.md

```diff
@@ -0,0 +1,27 @@
+---
+group: compliance
+title: Content Security Policy
```

Comment:
It's not easy to distinguish between this topic and the one in the PHP Guide. I recommend retitling this to "Content Security Policy Overview" and the other topic to something like "implement Content Security Policies".  I'm open to other options.

Also, you need some Algolia search markers. I'll look them up later.

Reply:
That sounds good.

Reply:
I changed the title and file name of my topic. It doesn't seem necessary to add "Implement" to your topic, because it's understood. But in any event, that's your call.